src/components/common/markets/clob-widget/clob-market-trade-form.tsx

```
@@ -0,0 +1,400 @@
+        totalCost += contractsToBuy * entry.price
+
+        remainingAmount -= contractsToBuy * entry.price
+
+        if (remainingAmount <= 0) break
+      }
+
+      const averagePrice = totalContracts > 0 ? totalCost / totalContracts : 0
+      const totalProfit = totalContracts * (1 - averagePrice)
+
+      return {
+        contracts: isNaN(totalContracts) ? 0 : totalContracts,
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)